components/tooltip/tooltip.js

```diff
@@ -183,9 +183,10 @@ class Tooltip extends RtlMixin(LitElement) {
 				--d2l-tooltip-outline-color: rgba(255, 255, 255, 0.32);
 				box-sizing: border-box;
 				color: white;
-				display: none;
+				display: inline-block;
 				position: absolute;
 				text-align: left;
+				visibility: hidden;
```

Comment:
Crazy that it was this simple, but makes sense!

The only side effect I can think of is that these tooltips are now going to show up when a screen reader user is just arrowing through the content of the page using the virtual cursor. For cases where these tooltips are linked to an interactive element via `aria-labelledby`/`aria-describedby` that's totally fine... but for the other scenarios it may not be ideal.

A potential solution to that might be for us to us to distinguish between these two cases -- `display: none` for non-interactive and `visibility: hidden` for interactive ones.

Reply:
Yep, I'll do some testing and check if that works

Reply:
After looking into this it should not be a concern, [visibility: hidden](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns) removes itself from the accessibility tree so it will not be read out.

Reply:
Great, thanks!